backend/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id 'org.springframework.boot' version '2.5.4'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'java'
+}
+
+group = 'com.pokemon'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+configurations {
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-jdbc'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    compileOnly 'org.projectlombok:lombok'
+    annotationProcessor 'org.projectlombok:lombok'
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    implementation 'com.graphql-java:graphql-java:16.2'
+    implementation 'com.google.guava:guava:26.0-jre'
```

Comment:
Also curious, having Spring also adding Guava seems too many, but just asking.

Reply:
It was my first time using graphql. I searched some documentation, tried them and just used the first one i 'completed' understood and com.graphql-java:graphql-java needed guava. Because of time i wasn't able to optimize more and check everything in detail